app/src/main/java/com/google/droidjump/MainActivity.java

```diff
@@ -35,23 +36,30 @@
 import com.google.android.gms.auth.api.signin.GoogleSignInResult;
 import com.google.android.gms.common.images.ImageManager;
 import com.google.android.gms.games.Games;
+import com.google.android.gms.games.LeaderboardsClient;
 import com.google.android.gms.games.Player;
 import com.google.android.gms.games.PlayersClient;
+import com.google.android.gms.games.leaderboard.LeaderboardScore;
+import com.google.android.gms.games.leaderboard.LeaderboardVariant;
 import com.google.android.gms.tasks.OnFailureListener;
 import com.google.android.material.navigation.NavigationView;
 import com.google.droidjump.models.LevelManager;
 import com.google.droidjump.models.NavigationHelper;
+import java.util.Timer;
+import java.util.TimerTask;
 
 /**
  * Represents main activity.
  */
 public class MainActivity extends FragmentActivity {
     // Request code used to invoke sign in user interactions.
     private static final int RC_SIGN_IN = 9001;
+    private static final int MINUTE_IN_MILLISECONDS = 60000;
```

Comment:
nit: usually we write it like "60 * 1000" to make it visible that minute is really 60 seconds and second is really 1000 milliseconds.